### PR TITLE
fix/PD-1676

### PR DESCRIPTION
**Pull request by CarlaCostea**
https://illuminate.atlassian.net/browse/PD-1676
-added fix to prevent crash

### CODE REVIEW

File: packages/editable-html/src/serialization.jsx [hunk 1]
```diff
@@ -356,7 +356,13 @@ serializer.deserialize = function deserialize(html) {
   return null;
 };
 
-export const htmlToValue = html => serializer.deserialize(html);
+export const htmlToValue = html => {
+  try {
+    return serializer.deserialize(html);
+  } catch {
+    return {};
```
Comment (andreeapescar) [inline packages/editable-html/src/serialization.jsx:363]: @CarlaCostea can you please also log the error?

File: packages/editable-html/src/index.jsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ export default class EditableHtml extends React.Component {
     const v = htmlToValue(props.markup);
     const current = htmlToValue(this.props.markup);
 
-    if (!v.equals(current)) {
+    if (v.equals && !v.equals(current)) {
```
Comment (andreeapescar) [inline packages/editable-html/src/index.jsx:50]: Why do you need this? Is there a point when this function does not exist?

File: packages/editable-html/src/serialization.jsx (hunk 1 above)
Comment (CarlaCostea) [reply packages/editable-html/src/serialization.jsx:363]: sure

File: packages/editable-html/src/index.jsx (hunk 2 above)
Comment (CarlaCostea) [reply packages/editable-html/src/index.jsx:50]: yes, we need this in the case that` serializer.deserialize(html)` does not happen

**Pull request merged**